.browserslistrc

```diff
@@ -0,0 +1,11 @@
+>= 1%
+last 1 major version
+not dead
+Chrome >= 45
+Firefox >= 38
+Edge >= 12
+Explorer >= 10
```

Comment:
We don't support IE10...

Reply:
Who gets prefixed CSS properties on the documentation page is not nessessarily the same set as who gets to run the the test runner

Reply:
True, but...
> On January 12, 2016, support ended for IE10 on Windows operating systems capable of running Internet Explorer 11

Reply:
Since IE11 has been available since Nov 2013, I just don't see the point...
Since Jan 2016, only the latest version of the browser still gets security updates...

Reply:
Suggest your browser ranges then. This is kind of what this file can help us do

Reply:
* Safari 11+
* IE11 only
* others: most recent LTS, and 2 most recent majors

But migrate contents to "package.json" and remove this file...

Reply:
I disagree on moving this to package.json. Eslint can also be configured in package.json but we don't do that. There are tonnes of configuration files in the root. Why is this one a problem?

Reply:
Suggested [what they did](https://github.com/browserslist/browserslist#queries)...

Reply:
Changed the browser ranges to reflect more modern browsers

Reply:
BTW, my apologies at having added to the confusion here. Many of the comments really belonged to runtime aspect, not website generation.

I'll blame not thinking straight on a lack of coffee.